docs/php/symfony/storyblok/webhook.mdx

```diff
@@ -0,0 +1,129 @@
+# Webhooks
+
+The Storyblok bundle automatically integrates in to all webhooks that are sent from Storyblok. It parses the requests, validates it and passes properly typed events with payloads to your code, to integrate into your app.
+
+## Installation
+
+After setting up the bundle routes[^1], you need to configure the webhook URL in Storyblok:
+
+```txt
+https://.../storyblok/webhook
+```
```

Comment:
```suggestion
After setting up the bundle routes[^1], you need to configure the webhook URL in Storyblok:

```txt
https://.../storyblok/webhook
```

Make sure to **not** add an extra trailing slash, otherwise the request will fail.
```

Reply:
We could probably put that in a warning-box. I forgot the macro for that, though. Will look it up, if you find this information useful.

Reply:
This was now fixed in the code: you can add the slash now.